<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1373</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1111</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>10</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>图像采集</string>
    </property>
    <widget class="QPushButton" name="pushButton_xuanze">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>选择工艺</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>50</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>   WPS号：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>开始采集</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>停止采集</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>140</y>
      <width>241</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab{width:75}
QTabBar::tab{height:25}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>缺陷检测</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>233</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>分类检测</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>211</width>
         <height>83</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>下塌检测</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>飞溅检测</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>清理缓存</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>231</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>缺陷统计</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>飞溅数量：</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>飞溅面积：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>  %  </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>下塌数量：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_5"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>下塌面积：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>  %  </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>370</y>
        <width>231</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>结果保存</string>
      </property>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>合格检测</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>209</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>保存检测结果</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>飞溅数据</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>235</width>
        <height>481</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>面积</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>周长</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>圆形度</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>下塌数据</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>235</width>
        <height>481</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>面积</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>周长</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>圆形度</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1373</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_7">
     <property name="title">
      <string>最近打开</string>
     </property>
     <addaction name="qingchujilu"/>
    </widget>
    <addaction name="dakai"/>
    <addaction name="menu_7"/>
    <addaction name="guanbitu"/>
    <addaction name="baocun"/>
    <addaction name="newbaocun"/>
    <addaction name="tuichu"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>简单变换</string>
    </property>
    <addaction name="xuanzhuan90"/>
    <addaction name="xuanzhuan180"/>
    <addaction name="fangda"/>
    <addaction name="suoxiao"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>工艺信息录入</string>
    </property>
    <addaction name="djgluru"/>
    <addaction name="fhluru"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>焊缝缺陷检测</string>
    </property>
    <addaction name="Tyuchuli"/>
    <addaction name="Tfenge"/>
    <addaction name="Tshibie"/>
    <addaction name="xiata"/>
    <addaction name="jcbaocun"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>系统管理</string>
    </property>
    <addaction name="shujuku"/>
    <addaction name="bangzhu"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="dakai"/>
   <addaction name="baocun"/>
   <addaction name="guanbitu"/>
   <addaction name="separator"/>
   <addaction name="fangda"/>
   <addaction name="suoxiao"/>
   <addaction name="xuanzhuan90"/>
   <addaction name="xuanzhuan180"/>
   <addaction name="separator"/>
   <addaction name="Tyuchuli"/>
   <addaction name="Tfenge"/>
   <addaction name="Tshibie"/>
   <addaction name="jcbaocun"/>
   <addaction name="separator"/>
   <addaction name="yijianhuanyuan"/>
   <addaction name="separator"/>
   <addaction name="shujuku"/>
   <addaction name="bangzhu"/>
   <addaction name="jiance"/>
  </widget>
  <action name="dakai">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/open.png</normaloff>:/zhujiemian/image/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="baocun">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/save.png</normaloff>:/zhujiemian/image/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="guanbitu">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/close.png</normaloff>:/zhujiemian/image/close.png</iconset>
   </property>
   <property name="text">
    <string>关闭图片</string>
   </property>
  </action>
  <action name="xuanzhuan90">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/rotate90.png</normaloff>:/zhujiemian/image/rotate90.png</iconset>
   </property>
   <property name="text">
    <string>旋转90度</string>
   </property>
  </action>
  <action name="xuanzhuan180">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/rotate180.png</normaloff>:/zhujiemian/image/rotate180.png</iconset>
   </property>
   <property name="text">
    <string>旋转180度</string>
   </property>
  </action>
  <action name="fangda">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/zoomin.png</normaloff>:/zhujiemian/image/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="suoxiao">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/zoomout.png</normaloff>:/zhujiemian/image/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="Tyuchuli">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/pretreat.png</normaloff>:/zhujiemian/image/pretreat.png</iconset>
   </property>
   <property name="text">
    <string>图像预处理</string>
   </property>
  </action>
  <action name="Tfenge">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/spilt.png</normaloff>:/zhujiemian/image/spilt.png</iconset>
   </property>
   <property name="text">
    <string>图像分割提取</string>
   </property>
  </action>
  <action name="Tshibie">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/final.png</normaloff>:/zhujiemian/image/final.png</iconset>
   </property>
   <property name="text">
    <string>焊缝缺陷识别</string>
   </property>
  </action>
  <action name="qingchujilu">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/clear.png</normaloff>:/zhujiemian/image/clear.png</iconset>
   </property>
   <property name="text">
    <string>清除记录</string>
   </property>
  </action>
  <action name="yijianhuanyuan">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/reduction.png</normaloff>:/zhujiemian/image/reduction.png</iconset>
   </property>
   <property name="text">
    <string>一键还原</string>
   </property>
  </action>
  <action name="jcbaocun">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/record.png</normaloff>:/zhujiemian/image/record.png</iconset>
   </property>
   <property name="text">
    <string>检测结果保存</string>
   </property>
  </action>
  <action name="shujuku">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/data.png</normaloff>:/zhujiemian/image/data.png</iconset>
   </property>
   <property name="text">
    <string>数据维护</string>
   </property>
  </action>
  <action name="bangzhu">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/help.png</normaloff>:/zhujiemian/image/help.png</iconset>
   </property>
   <property name="text">
    <string>系统帮助</string>
   </property>
  </action>
  <action name="tuichu">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/exit.png</normaloff>:/zhujiemian/image/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="djgluru">
   <property name="text">
    <string>单激光焊接</string>
   </property>
  </action>
  <action name="fhluru">
   <property name="text">
    <string>激光电弧复合焊接</string>
   </property>
  </action>
  <action name="jiance">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/stock-color-brightness-contrast.png</normaloff>:/zhujiemian/image/stock-color-brightness-contrast.png</iconset>
   </property>
   <property name="text">
    <string>监测系统</string>
   </property>
  </action>
  <action name="xiata">
   <property name="text">
    <string>下塌检测</string>
   </property>
  </action>
  <action name="newbaocun">
   <property name="icon">
    <iconset resource="ziyuan.qrc">
     <normaloff>:/zhujiemian/image/paste.png</normaloff>:/zhujiemian/image/paste.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ziyuan.qrc"/>
 </resources>
 <connections/>
</ui>
